server/mergin/auth/schemas.py

```diff
@@ -33,17 +33,16 @@ def get_disk_usage(self, obj):
 
     def _has_project(self, obj):
         # DEPRECATED functionality - kept for the backward-compatibility
-        from ..sync.models import Project, ProjectAccess
+        from ..sync.models import ProjectUser, Project
 
         ws = current_app.ws_handler.get_by_name(obj.user.username)
         if ws:
             projects_count = (
-                Project.query.filter(Project.creator_id == obj.user.id)
+                Project.query.join(ProjectUser)
+                .filter(Project.creator_id == obj.user.id)
```

Comment:
creator_id looks like new condition here. Do we need this check?

Reply:
creator_id was there before in filter, but in fact it is some legacy stuff for backward compatibility

Reply:
ok ... this looks like legacy method :)